tasks/natural-language-processing/sentiment-analysis/sentiment_analysis_model.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-

```

Comment:
Just Noticed that I gave comment on file which is located on root of the ai-platform repository. Please consider above comments for this code and remove above file. I believe you did not use git mv command to move it into appropriate directory otherwise you would not have it in both places.